pandas/core/generic.py

```diff
@@ -4873,13 +4874,22 @@ def sample(
         falcon    2
         Name: num_legs, dtype: int64
 
-        A random 50% sample of the ``DataFrame`` with replacement:
+        A random 50% sample of the ``DataFrame``:
 
-        >>> df.sample(frac=0.5, replace=True, random_state=1)
+        >>> df.sample(frac=0.5, random_state=1)
               num_legs  num_wings  num_specimen_seen
         dog          4          0                  2
         fish         0          0                  8
 
+        Extract 4 random elements with replacement:
+
+        >>> df.sample(n=4, replace=True, random_state=1)
```

Comment:
doctest failure: https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=14665

Do you get this output locally? You may need to play with random_state to get an outcome we want.

Reply:
Yes, that's what I got when playing around with the example data frame in `sample()`. I purposefully kept the same `random_state=1` and increased `n` until I got a dup. (If you'd rather only 2 rows, use `random_state=4` with `n=2`.)

Reply:
Ah, sorry it's actually L4879 (the `df.sample(frac=0.5, random_state=1)`) that needs to be updated now that `replace=False` there